package.json

```diff
@@ -9,8 +9,8 @@
     "build:exchangerates": "node scripts/exchangerates.js",
     "build:babel": "babel src --compact=true -s --out-dir dist",
     "build:webpack": "webpack --optimize-minimize --optimize-dedupe",
-    "watch:js": "npm-run-all --parallel watch:webpack watch:babel",
-    "watch:babel": "npm run build:babel -- -w -q",
+    "watch:js": "npm-run-all --parallel watch:babel watch:webpack",
```

Comment:
Not really, but webpack listens for changes in babel, and webpack does an initial build, along with babel. So it just felt logical to put them in their logical order.

Also with this order, it more likely to reduce log outputs that and not needed, but that's only on average.